libmscore/scale.h

```diff
@@ -0,0 +1,50 @@
+//=============================================================================
+//  MuseScore
+//  Music Composition & Notation
+//
+//  Copyright (C) 2002-2012 Werner Schweer
+//
+//  This program is free software; you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License version 2
+//  as published by the Free Software Foundation and appearing in
+//  the file LICENCE.GPL
+//=============================================================================
+
+#ifndef __SCALE_H__
+#define __SCALE_H__
+
+/**
+ \file
+ Definition of Scale class.
+ */
+
+#include "note.h"
+
+namespace Ms {
+
+class Scale {
+public:
+      bool loadScale(const QString&);
+      double getTuning(const Note* note) const;
+
+private:
+      QString getNextLine(QTextStream& in);
+
+      std::vector<double> originalNotes;
```

Comment:
"_If you find there is a case for having a sharp note higher than the next bemol, you could always change the scala file and input it._".

This would no longer be a valid Scala file. I don't understand why we should force a defined format beyond its intended usage, when a more adequate one (perfectly adequate, I would say) is easy (easier, I would say) to define and manage.

Scala format should be kept ad an import option, given the amount of data available under it. But I maintain it is not adequate as internal data structure.